docs/fitdla.rst

```diff
@@ -40,6 +40,7 @@ script `pyigm_fitdla`.  Here is the usage::
                             Input DLA Fit file
       --conti_file CONTI_FILE
                             Input continuum spectrum
+                            Option 'from_spectrum': continuum data will be used from the original (in_file) spectrum.
```

Comment:
Add that 100 spline points will be added uniformly 
along the input continuum.

Reply:
Added.